static/js/app.js

```diff
@@ -847,6 +870,95 @@ $(function () {
             this.fetchMarkers();
             this.updateFilterString();
         },
+        loadFilterFromParameters: function() {
+            var bool_atrs = {};
+            bool_atrs["show_discussions"] = this.showDiscussion;
+            bool_atrs["show_markers"] = this.show_markers;
+            bool_atrs["accurate"] = this.accurate;
+            bool_atrs["approx"] = this.approx;
+            bool_atrs["show_fatal"] = this.show_fatal;
+            bool_atrs["show_severe"] = this.show_severe;
+            bool_atrs["show_light"] = this.show_light;
+
+            $.each(bool_atrs, function(attr, attr_value) {
+                if (attr_value == '1'){
+                    $(attr).prop("checked", true);
+                }else{
+                    $(attr).prop("checked", false);
+                }
+            });
+
+            var complex_attrs = {};
+            complex_attrs["show_urban"] = this.show_urban;
+            complex_attrs["show_intersection"] = this.show_intersection;
+            complex_attrs["show_lane"] = this.show_lane;
+
+            $.each(complex_attrs, function(attr, attr_value) {
+                if (attr_value == 3){
+                     $(attr).prop(":checked",true);
+                     $(attr).prop(":checked", true);
+                }else if (attr_value == 2){
+                     $(attr).prop(":checked", true);
+                     $(attr).prop(":checked", false);
+                }else{
+                     $(attr).prop(":checked", false);
+                     $(attr).prop(":checked", false);
+                }
+            });
+
+            $("input[type='radio'][name='weather']").each(function() {
```

Comment:
Not working. I tried 

$.each(radio_attrs, function(attr, attr_value) {
                $("input[type='radio'][name='" + attr +"']").each(function() {
                    if($(this).val() == attr_value) {
                        $(this).prop("checked", true);
                    }
                });
            });

but it also doesn't work properly.